Mathlib/Data/Real/StandardPart.lean

```diff
@@ -0,0 +1,320 @@
+/-
+Copyright (c) 2025 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+module
+
+public import Mathlib.Algebra.Order.Ring.Archimedean
+public import Mathlib.Algebra.Ring.Subring.Order
+public import Mathlib.Data.Real.Archimedean
+public import Mathlib.Data.Real.CompleteField
+public import Mathlib.Order.Quotient
+public import Mathlib.RingTheory.Valuation.ValuationSubring
+
+/-!
+# Standard part function
+
+Given a finite element in a non-archimedean field, the standard part function rounds it to the
```

Comment:
I get why I am totally confused now.
Perhaps you should say "non-archimedean ordered field" because to me (and probably to lots of other people) non-archimedean field probably means [`NonarchimedeanRing`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Nonarchimedean/Basic.html#NonarchimedeanRing) + field.